docs/stylesheets/reference.css

```
@@ -0,0 +1,30 @@
+.awa-status {
+  display: inline-block;
+  white-space: nowrap;
+  padding: 0.12rem 0.45rem;
+  border: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)